doc/EnvironmentalMonitoring/Monitoring-messages.md

```diff
@@ -33,6 +34,13 @@ This document is licensed under the Creative Commons Attribution-ShareAlike 3.0
 
 Fetching security server metrics uses the X-Road protocol. The `getSecurityServerMetrics` request requires a `securityServer` header element as specified by the security server targeting extension for the X-Road message protocol \[[PR-TARGETSS](#Ref_PR-TARGETSS)\] so that the request can be routed to a specific security server.
 
+`Body` element must contain `getSecurityServerMetrics` element. 
+
+Optional `outputSpec` element can be used to define a set of specific environmental monitoring parameters. In this case a set of `outputField` elements are used to define requested metrics. 
+`outputField` elements should match metric names provided by [Dropwizard Metrics](https://github.com/dropwizard/metrics). Empty or missing `outputSpec` element will request all metrics. `getSecurityServerMetricsResponse` element in response will contain matching metrics.
```

Comment:
That is really misleading. If you want to ask for certificates then you must use `<m:outputField>Certificates</m:outputField>`, but Monitoring-architecture.md contains information about `CertificateInfoSensor`, and https://github.com/dropwizard/metrics leads to some github repository that probably would not tell anything about `Certificates` metric. It would be great to list all available metrics or at least write that "you should make a query without `outputField` to find the list of all available metrics".

Reply:
Agreed, reworded and should be more clear now.